lonboard/_layer.py

```diff
@@ -262,6 +278,14 @@ class BaseArrowLayer(BaseLayer):
     # The following traitlets **are** serialized to JS
 
     table: traitlets.TraitType
+    selected_bounds: traitlets.Tuple(
+        traitlets.Float(),
+        traitlets.Float(),
+        traitlets.Float(),
+        traitlets.Float(),
+        allow_none=True,
+        default_value=None,
+    ).tag(sync=True)
```

Comment:
This doesn't need to be on the BaseArrowLayer as well. It will inherit from `BaseLayer`.